src/utils/normalizeWheel.ts

```diff
@@ -42,6 +43,12 @@ function normalizeWheel(event: any) {
     pX = event.deltaX;
   }
 
+  // on some platforms (e.g. Win10), browsers do not automatically swap deltas for horizontal scroll
+  if (!isSideScrolling && !event.ctrlKey && event.shiftKey && pX === 0) {
```

Comment:
Adding `!isSideScrolling` to try and limit any potential of the change breaking existing behavior